### PR TITLE
[issue-44]: Create generated bug template

### LINKED ISSUE #44: ci: Create pre-generated templates for issue types
We should be prepared for the floodgate of opened issues opened by us and others as the development progresses after time

I do like the idea what tendermint has in its main repo and it will help us to automate the triaging process for the future tickets
Here is the list of initial templates(could be more in the future if needed) :

1. Bug
2. Feature Request
3. General Questions
4. Change Request for existing feature
5. Others

Detailed information for each of the point:
No. 1 - Similar to tendermint's [bug template](https://github.com/tendermint/tendermint/issues/new?assignees=&labels=&template=bug-report.md)
No. 2 - If the user can see that a new feature is beneficial for the product, he/she/they can propose it here
No. 3 - StackOverflow like questions specific to the repo (can be moved to another place if neccessary)
No. 4 - If the user finds out that the existing <ABC> feature should be reworked/changed/improved using <XYZ> lib/architecture, then he/she/they can do it here 
No. 5 - if none of 4 above fits the user's need

For now, it seems like No. 2 and No. 4 can be merged into 1 template. Nevertheless, as the product grows, it will be hard and  intense in manual human-labour for the team/contributors to triage what requests are "freshly new features" and what requests are "outdated/should be updated"

### PR DESCRIPTION
**Pull request by Bidon15**
Create a bug template as a form
Create config.yml file to disallow creation of blank issues
Notes:
1. Still config.yml will be set to true before creating a separate template for feature requests
2. A code of conduct is parked in issue #75 for further discussions

Closes #44

### CODE REVIEW

**Comment by Bidon15** [comment]
To play around with issues, please navigate[ to my test repo ](https://github.com/Bidon15/gogitact/issues/new/choose)
Screenshot 2021-09-22 at 13 37 57
Screenshot 2021-09-22 at 13 38 39
Screenshot 2021-09-22 at 13 39 22
Screenshot 2021-09-22 at 13 39 02
Screenshot 2021-09-22 at 13 39 40

File: .github/ISSUE_TEMPLATE/general-question.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: General Question
```
Comment (liamsi) [inline .github/ISSUE_TEMPLATE/general-question.yml:1]: An alternative to questions could be using [discussions](https://docs.github.com/en/discussions)? Just an idea.

Comment (renaynay) [reply .github/ISSUE_TEMPLATE/general-question.yml:1]: Yes I think we should remove the `General Question` option and redirect to discussions or something.

**Review by renaynay — APPROVED** [review]
LGTM, thank you !!

File: .github/ISSUE_TEMPLATE/general-question.yml (hunk 1 above)
Comment (Bidon15) [reply .github/ISSUE_TEMPLATE/general-question.yml:1]: Damn, I forgot about this feature 😅 
Yeap. Let's remove the General Question and have discussions

**Review by Wondertan — APPROVED** [review]
Nice!

**Pull request merged**